benchmark-overhead/src/test/java/io/opentelemetry/results/CsvPersister.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package io.opentelemetry.results;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Comparator;
+import java.util.List;
+import java.util.function.Consumer;
+
+class CsvPersister implements ResultsPersister {
+
+  private final Path resultsFile;
+
+  public CsvPersister(Path resultsFile) {this.resultsFile = resultsFile;}
+
+  @Override
+  public void write(List<AppPerfResults> results) {
+
+    ensureFileCreated(results);
+
+    StringBuffer sb = new StringBuffer().append(System.currentTimeMillis() / 1000);
```

Comment:
you probably meant to use `StringBuilder`

Reply:
I did.  :) Thank you.